### ISSUE TITLE
Feature: A way to share power-mode configurations

### ISSUE THREAD

**Issue by ayrtonmassey** [hoovercj/vscode-power-mode#5]
It would be great to have some way of packaging & sharing:

- My custom configuration settings for power-mode.
- The custom images I am using, if they are local files (not web URLs).

**Comment by hoovercj** [comment]
I think the easiest solution for now would be to set up a dedicated issue thread for this where people can leave comments of the form:

```
{
    "Label": "Clippy",
    "Description": "Clippy follows along next to your cursor",
    "Example": "https://path/to/gif",
    "Settings": {}
}
```

The extension can query the github api to get the comments and parse the settings out of them.

These can be presented either as completions in settings.json or some other way such as commands.

Example api url:

I could totally do comments for now: https://api.github.com/repos/hoovercj/vscode-power-mode/issues/5/comments

**Comment by hoovercj** [comment]
This is now published in v1.2.0
